kikuchipy/signals/util/tests/test_dask.py

```diff
@@ -38,18 +40,21 @@ def test_chunk_shape(self):
         chunks = get_chunking(s, chunk_shape=16)
         assert chunks == ((16, 16), (16, 16), (256,), (256,))
 
+    @pytest.mark.skipif(sys.version_info < (3, 8), reason="Requires Python >= 3.8")
```

Comment:
Skip with dask version that this test doesn't support?

Reply:
See https://github.com/pyxem/kikuchipy/pull/567#discussion_r985495423.